gui/widgets/bandview.cpp

```diff
@@ -106,6 +143,17 @@ void BandView::updateLabeling(const cv::Mat1s &newLabels, const cv::Mat1b &mask)
 	refresh();
 }
 
+
```

Comment:
Why is the override of mouseMoveEvent() needed when it just calls ScaledView? Shouldn't we remove the override instead?